docs/guide/webpack.md

```diff
@@ -338,6 +340,43 @@ If a custom bundle imports packages that are also imported by the "main" bundle
 
 Apostrophe will take care of deploying the output files of custom bundles alongside those generated by the main bundle. Apostrophe will also generate the needed `script` and `link` tags automatically.
 
+### Rebundling resources
+
+In some cases, an npm package will provide front-end code that is either unbundled or bundled in a way that doesn't reflect the needs of the project. In these cases, the bundle configuration set by each module can be overridden through the `rebundleModules` option of the `@apostrophecms/asset` module.
+
+This option takes an object with module or module file names as properties and bundle names as values.
+
+<AposCodeBlock>
+
+```js
+module.exports = {
+  options: {
+    rebundleModules: {
+      // Everything from the fancy-form module should go in the regular "main" bundle
+      'fancy-form': 'main',
+      // Everything from the basic-product module should go in the "secondary" bundle
+      'basic-product': 'secondary',
+      // The ui/src/form.js code from the @dcad/form module should be retargeted to the
+      // "secondary" bundle — but only that code, leave ui/src/index.js in the main bundle
+      '@dcad/form:form': 'secondary'
+    }
+  }
+};
+```
+
+<template v-slot:caption>
+modules/@apostrophecms/asset/index.js
+</template>
+</AposCodeBlock>
+
+If the module name is given, without specifying a file name, all of the JavaScript assets from the `module/ui/src` folder will be added to the named bundle. If a file name is provided by appending it to the module name with a `:`, then only that file will be added to the named bundle. Other assets in the same folder, for example, `index.js`, will be added to the bundle specified by the module.
+
+The module name for project-level modules, like `modules/@apostrophecms/home-page` should be `@apostrophecms/my-home-page`. To target the original (npm) module use: `@apostrophecms/home-page`.
```

Comment:
To avoid ambiguity, when using `rebundleModules` to move assets provided as part of the project-level configuration of an apostrophe module originally loaded from npm, such as code found in `modules/@apostrophecms/home-page/ui/src/index.js`, use the module name `@apostrophecms/my-home-page`. Without the `my-` prefix the code from the original npm module would be moved.

Reply:
Changed.